toolkit/resources/imageconfigs/packagelists/iso-initrd-packages-arm64.json

```diff
@@ -0,0 +1,66 @@
+{
+    "packages": [
```

Comment:
Are there any big differences between the AMD64 version? If there are non, let's use one JSON and if they have many packages in common, let's put the common part into one JSON and whatever differs into separate ones. This way it's going to be easier to understand the special requirements for each architecture.

Reply:
There are differences like calamares, grub2-pc etc which are not available for Arm64 arch. grub2-pc is required for x64 BIOS based systems.